package.json

```diff
@@ -32,7 +32,12 @@
     "minimist": "^1.2.5",
     "napa": "^3.0.0",
     "popper.js": "^1.16.1",
-    "replace-in-file": "^5.0.2"
+    "replace-in-file": "^5.0.2",
+    "stylelint": "^13.3.3",
+    "stylelint-config-sass-guidelines": "^7.0.0",
+    "stylelint-config-standard": "^20.0.0",
+    "stylelint-order": "^4.0.0",
+    "stylelint-scss": "^3.17.1"
```

Comment:
Does the lint stuff need to be built within the OpenEMR package?

Reply:
btw, not a big deal if needed since the node_modules directory is removed for production packages anyways. More just curious how we run these processes/checks.

Reply:
@bradymiller From a developer standpoint, this isn't like php codesniffer where you could add it in with Travis CI. Most people like myself would actually use this as a guide to make sure they for example, put the class attributes in ABC order. The only way to make that work is to have it already added in so people know what plugins we have added and can utilize all the styling standards just by using a single command.